src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,16 @@
+import { MovieCard } from '../MovieCard';
 import './MovieList.scss';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MovieList = ({ movies }) => (
+  <>
+    <div className="movies">
```

Comment:
Remove redundant react fragment and div wrapper

https://www.memberstack.com/blog/react-fragment

Reply:
Please fix comment.  If you have any questions, please ask them in fe_chat and tag mentors. It will be faster, because your answers in GitHub will be visible to mentor only after you re-request review